qiskit_machine_learning/kernels/fidelity_statevector_kernel.py

```diff
@@ -40,7 +40,8 @@ class FidelityStatevectorKernel(BaseKernel):
 
         K(x,y) = |\langle \phi(x) | \phi(y) \rangle|^2.
 
-    In this implementation, :math:`|\phi(y)\rangle` is given by a ``Statevector.data`` array. These
+    In this implementation, :math:`|\phi(y)\rangle` is given by a
+    :attr:`qiskit.quantum_info.Statevector.data` array. These
```

Comment:
> is given by a :attr:`qiskit.quantum_info.Statevector.data`

Well yes and no. AerStatevector can also be used here https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.quantum_info.AerStatevector.data.html 

Maybe if it says something like

> is given by a :attr:`qiskit.quantum_info.Statevector.data` array or the same field of a compatible sub-class

My concern is here in making it look too specific to the subclass by having the specific link which might lead to the belief that only that can be used. The parameter itself does have a note about StateVector

Reply:
Right. Because AerStatevector is a subclass of Statevector, I've replaced this sentence with the following:

> In this implementation, :math:`|\phi(y)\rangle` is given by the ``data`` attribute of a :class:`~qiskit.quantum_info.Statevector` object or one of its subclasses.